### ISSUE TITLE
WCAG Review: Products Accessibility Round-up

### ISSUE THREAD

**Issue by dan-bowerman** [ramp4-pcar4/story-ramp#426]
WCAG re-review was performed on all Storylines products to date. A few minor issues; all appear to be content related and easily resolved. The following links are all internal and should be fixed in-place before committed to source control. 

- [x] [Climate Change](https://environmental-maps.dev.ec.gc.ca/ramp-storylines_sandbox/index-ca-en.html#/en/08d9bd86-6535-4daa-8284-c5ade064982c)
2 violations: Slide 7 & 8, empty header: `<h2 data-v-abf648e2="""" class=""px-10 mb-0 chapter-title top-20"" data-scrollama-index=""0""></h2>`

- [x] [Particulate Matter](https://environmental-maps.dev.ec.gc.ca/ramp-storylines_sandbox/index-ca-en.html#/en/ef6674bd-f2f8-4327-a8b4-77aa5620603a)
1 Violation: Intro image, missing alt-text `<img class=""inline-block"" src=""ef6674bd-f2f8-4327-a8b4-77aa5620603a/assets/en/NPRIpictogramme-2016data-EN.jpg"">`

- [x] [Metal Ore Mining](https://environmental-maps.dev.ec.gc.ca/ramp-storylines_sandbox/index-ca-en.html#/en/9babb72d-e613-4b04-9385-c8454120c59c)
1 Violation: Missing context link `<footer class=""p-8 pt-2 text-right text-sm"">Context:<a class=""text-blue-700 font-semibold"" href="""" target=""_NEW""></a>`

- [x] [Arsenic](https://environmental-maps.dev.ec.gc.ca/ramp-storylines_sandbox/index-ca-en.html#/en/113v89fa-6id7-7258-442e-7a86f29w1e5k)
2 Violations:
   - Chapter navigation, missing context link: 
```<a data-v-87a4c248="""" aria-current=""page"" href=""#/en/113v89fa-6id7-7258-442e-7a86f29w1e5k#15-"" class=""router-link-active router-link-exact-active flex py-1 px-3"" target=""""><svg data-v-87a4c248="""" class=""flex-shrink-0"" width=""0"" height=""0"" viewBox=""0 0 24 24"" xmlns="http://www.w3.org/2000/svg" fill=""#fff"" stroke=""#878787""><path data-v-87a4c248="""" d=""m19.325 16.229c-2.4415 1.4096-4.8829 2.8191-7.3244 4.2286-2.4415-1.4096-4.883-2.8192-7.3245-4.2288-3.55e-5 -2.8191-7.1e-5 -5.6383-1.066e-4 -8.4574 2.4415-1.4096 4.8829-2.8191 7.3244-4.2286 2.4415 1.4096 4.883 2.8192 7.3245 4.2288 3.7e-5 2.8191 7.4e-5 5.6383 1.1e-4 8.4574z"" stroke-width="".93974""></path></svg><span data-v-87a4c248="""" class=""flex-1 ml-4 overflow-hidden leading-normal overflow-ellipsis whitespace-nowrap""></span></a>```
  - Chapter navigation, missing context link 
```<a data-v-689fab2c="""" aria-current=""page"" href=""#/en/113v89fa-6id7-7258-442e-7a86f29w1e5k#15-"" class=""router-link-active router-link-exact-active flex items-center px-2 py-1 mx-1"" target=""""><svg data-v-689fab2c="""" class=""flex-shrink-0"" width=""24"" height=""24"" viewBox=""0 0 24 24"" xmlns="http://www.w3.org/2000/svg" fill=""#fff"" stroke=""#878787""><path data-v-689fab2c="""" d=""m19.325 16.229c-2.4415 1.4096-4.8829 2.8191-7.3244 4.2286-2.4415-1.4096-4.883-2.8192-7.3245-4.2288-3.55e-5 -2.8191-7.1e-5 -5.6383-1.066e-4 -8.4574 2.4415-1.4096 4.8829-2.8191 7.3244-4.2286 2.4415 1.4096 4.883 2.8192 7.3245 4.2288 3.7e-5 2.8191 7.4e-5 5.6383 1.1e-4 8.4574z"" stroke-width="".93974""></path></svg><span data-v-689fab2c="""" class=""flex-1 ml-4 overflow-hidden leading-normal overflow-ellipsis whitespace-nowrap""></span></a>```

- [x] [Copper](https://environmental-maps.dev.ec.gc.ca/ramp-storylines_sandbox/index-ca-en.html#/en/55918bed-b04f-4d27-804b-7b62d44b77aa)
1 Violation: Slide 0, empty link `<a href="http://"></a>`

Full details are available in the attached review: 
[STB Storylines-2024-04-04-Assessment_WCAG_AA2.1.xlsx](https://github.com/ramp4-pcar4/story-ramp/files/14995911/STB.Storylines-2024-04-04-Assessment_WCAG_AA2.1.xlsx)
